source/_components/sensor.glances.markdown

```diff
@@ -32,6 +32,16 @@ $ curl -X GET http://IP_ADDRESS:61208/api/2/mem/free
 
 For details about auto-starting `glances`, please refer to [Start Glances through Systemd](https://github.com/nicolargo/glances/wiki/Start-Glances-through-Systemd).
 
+To enable a glances server e.g. on Ubuntu edit `/etc/default/glances` this way:
+```shell
+# Default is to launch glances with '-s' option.
+DAEMON_ARGS="-w"
+
+# Change to 'true' to have glances running at startup
+RUN="true"
+```
```

Comment:
Instead of duplicating the Glances documentation, a link like for systemd is sufficient and we don't need to take care if the information are up-to-date.